packages/codemods/src/transforms/transform-grid-options-v31-2/replacements.ts

```diff
@@ -22,12 +22,6 @@ export const replacements: Array<CodemodObjectPropertyReplacement> = transformOb
       transformObjectListValue(
         transformObjectProperties({
           suppressCellFlash: migrateProperty('enableCellChangeFlash', invertOptionalBooleanValue()),
-          dndSource: removeProperty(
```

Comment:
https://www.ag-grid.com/javascript-data-grid/upgrading-to-ag-grid-32/#column-options 

dndSource and dndSourceOnRowDrag are reinstated (no longer deprecated).